### PR TITLE
Sort nested data by config.x.order rather than the assumed deprecated…

### LINKED ISSUE #32: Line node order should match settings.x/y.order for ordinal measures.
Line node order differs from x/y domain order because it's sorted regardless of `settings.x/y.order`.  Check out `line 7`:
```
 1    if(sublevel){
 2      this_nest.key(d => d[sublevel]);
 3      this_nest.sortKeys((a,b) => {
 4        return config.x.type === 'time' ? d3.ascending(new Date(a), new Date(b)) :
 5          config.x_dom ? d3.ascending(config.x_dom.indexOf(a), config.x_dom.indexOf(b)) :
 6          sublevel === config.color_by && config.legend.order ? d3.ascending(config.legend.order.indexOf(a), config.legend.order.indexOf(b)) :
 7          config.x.type === 'ordinal' || config.y.type === 'ordinal' ? naturalSorter(a,b) :
 8          d3.ascending(+a, +b);
 9      });
10    }
```
[example](http://imgur.com/G5IIJvy)
```
        x: {
            column: null, //set in syncSettings()
            type: null, //set in syncSettings()
            behavior: 'flex',
            tickAttr: null,
            order:
                ['Baseline'
                ,'Visit 1'
                ,'Visit 2'
                ,'Visit 3'
                ,'Early Termination'
                ,'End of Study']
        },
```

### PR DESCRIPTION
**Pull request by samussiah**
… config.x_dom.

Closes #32

### CODE REVIEW

**Comment by samussiah** [comment]
Merging into #40 rather than master to avoid setting up the build process again.

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "d3": ">= 3.0.0"
   },
   "scripts": {
-    "es5": "rollup --config",
+    "es5": "rollup --config && sed -i 's/d3$1/d3/' build/webcharts.js",
```
Comment (jwildfire) [inline package.json]: Not sure this is needed ... Is the fact that `rollup` creates an alias (`d3$1`) causing any problems?

File: rollup.config.js [hunk 2]
```diff
@@ -6,7 +6,11 @@ export default {
     dest: 'build/webcharts.js',
     format: 'umd',
     globals: {d3: 'd3'},
-    external: ['d3'],
+    external: (function() {
```
Comment (jwildfire) [inline rollup.config.js]: Is this needed? Is anything broken in the current build if we leave both `package.json` and `rollup.config.js` unchanged?

**Comment by samussiah** [comment]
@jwildfire , that little `sed` bit to replace the `d3$1` alias has been removed.  The `devDependencies` have been simplified as well.

**Pull request merged**